plutus-contract/src/Wallet/Emulator/Error.hs

```diff
@@ -47,3 +47,15 @@ throwInsufficientFundsError = throwError . InsufficientFunds
 
 throwOtherError :: Member (Error WalletAPIError) effs => Text -> Eff effs a
 throwOtherError = throwError . OtherError
+
+-- | An error thrown by chain index APIs
+newtype ChainIndexAPIError =
+    OtherChainIndexError Text
```

Comment:
Is this error thrown anywhere?

Reply:
We did add more chain states (mainly caches for fast queries) and GET endpoints for our project but they aren't presented here yet. Do you think it makes sense to pursue this direction?